src/components/energyLabel/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC } from 'react';
+
+import Text from '../text';
+import Flex from '../flex';
+import Box from '../box';
+
+type Efficiency = 'A' | 'B' | 'C' | 'D' | 'E' | 'F' | 'G';
+
+interface Props {
+  efficiency: Efficiency;
+}
+
+const EnergyLabel: FC<Props> = ({ efficiency }) => {
+  const colors: Record<Efficiency, `#${string}`> = {
+    A: '#4CA651',
+    B: '#54B646',
+    C: '#CAD143',
+    D: '#FEF050',
+    E: '#F1AE3D',
+    F: '#EE6E2D',
+    G: '#D02F26',
+  };
+
+  return (
+    <Flex height="20px" width="40px">
```

Comment:
Can we make use of our patterns for the sizes?

Reply:
actually he changed his mind and we go with 20px/md now